### PR TITLE
Merged Scatterplots

### PR DESCRIPTION
**Pull request by marioaks**
1) Merges diverging scatterplot branches. I went ahead and removed the changes I made to Scatterplot and County Point and instead sorted the array of CountyPoints. As long as new scatterplots aren't introduced or changed, this should be ok. 

2) Adds in the new Tooltip module & merges in the mobile tooltip styling defined in the scatter-plot branch

3) Serves table & scatterplot index files at the same time with parcel v2. Should be easy to add in a maps index file as well if it's convenient.

### CODE REVIEW

**Comment by alexwgraves** [comment]
This encapsulates all of the changes in #37, #38, and #39, right? If so can you close those PRs?

**Comment by marioaks** [comment]
> This encapsulates all of the changes in #37, #38, and #39, right? If so can you close those PRs?

Closed #38 and #39. Added 2 comments to #37. Feel free to close that one once you see them.

File: src/css/graph.css [hunk 1]
```diff
@@ -53,13 +71,31 @@
   width: 8px;
   height: 8px;
   border-radius: 50%;
+}
+.key-circle:not(:empty)::before {
```
Comment (alexwgraves) [inline src/css/graph.css:76]: Add a return here pls

File: src/js/classes/Graph.js [hunk 2]
```diff
@@ -60,19 +78,20 @@ class CountyPoint {
...
+      this.tooltipTriggerTargets.push(point);
       point.addEventListener("mouseenter", () => this.onMouseEnter());
       point.addEventListener("mouseleave", () => this.onMouseLeave());
     });
   }
 
   renderLine() {
-    // only draw line if we have two data points
-    if (this.data.length != 2) return;
+    // // only draw line if we have two data points
```
Comment (alexwgraves) [inline src/js/classes/Graph.js]: There's an extra `//` here

**Review by alexwgraves — COMMENTED** [review]
A couple tiny tiny comments but other than that looks good!

**Comment by alexwgraves** [comment]
Closes #36

**Pull request merged**